WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsKnownViewController.swift

```diff
@@ -28,12 +28,14 @@ final class CardReaderSettingsKnownViewController: UIViewController, CardReaderS
     /// Accept our viewmodel and listen for changes on it
     ///
     func configure(viewModel: CardReaderSettingsPresentedViewModel) {
-        guard let viewModel = viewModel as? CardReaderSettingsKnownViewModel else {
+        self.viewModel = viewModel as? CardReaderSettingsKnownViewModel
```

Comment:
I think this could be compacted a bit, something like:
```
guard viewModel = viewModel as? CardReaderSettingsKnownViewModel else {
    DDLogError(....)
}

viewModel.didUpdate = onViewModelDidUpdate
```